src/test/php/PHPMD/TextUI/CommandLineOptionsTest.php

```diff
@@ -702,11 +703,13 @@ public function testExcludeOption(): void
     /**
      * @param class-string $expectedClass
      * @throws Throwable
-     * @throws Throwable
      * @dataProvider dataProviderCreateRenderer
      */
     public function testCreateRenderer(string $reportFormat, $expectedClass): void
     {
+        require_once self::$filesDirectory . '/PHPMD/Test/Renderer/NamespaceRenderer.php';
+        require_once self::$filesDirectory . '/PHPMD/Test/Renderer/PEARRenderer.php';
```

Comment:
Some PHPCS code style preference 🤷 
```suggestion

        require_once self::$filesDirectory . '/PHPMD/Test/Renderer/PEARRenderer.php';
```